Author: Daniel Kobras <kobras@debian.org>
Description: Update documentation to clarify that pluginfo option is not meant
 to be understood by non-developers.
---
 test/dvcont.1 |    5 +++--
 test/dvcont.c |    2 +-
 2 files changed, 4 insertions(+), 3 deletions(-)

--- libavc1394.orig/test/dvcont.1
+++ libavc1394/test/dvcont.1
@@ -18,7 +18,7 @@
 .\" This manual page was written especially for Debian Linux. It is based
 .\" on dvcont's help output using help2man and manual hacking.
 .\"
-.TH DVCONT 1 "September 2002"
+.TH DVCONT 1 "October 2002" "dvcont"
 .SH NAME
 dvcont \- send control commands to DV cameras
 .SH SYNOPSIS
@@ -95,7 +95,8 @@ given in format HH:MM:SS:FF as above.
 Report the status of the device.
 .TP
 .B pluginfo
-Report available plugs.
+Report available plugs.  (This option is mainly intended for internal
+development.)
 .TP
 .B verbose
 Tell the program to tell you debug info.
--- libavc1394.orig/test/dvcont.c
+++ libavc1394/test/dvcont.c
@@ -65,7 +65,7 @@ printf("\neject - Tell the camera to eje
 printf("\ntimecode - Report the timecode from the tape (HH:MM:SS:FF)");
 printf("\nseek - Tell the camera to go to the <timecode> (HH:MM:SS:FF)");
 printf("\nstatus - Report the status of the device");
-printf("\npluginfo - Report available plugs");
+printf("\npluginfo - Report available plugs (developers only)");
 printf("\nverbose - Tell the program to tell you debug info.");
 printf("\nversion - Tell the program to tell you the program version.");
 printf("\nhelp - Tell the program to show you this screen");
